README.md

```diff
@@ -27,7 +27,8 @@ $ make uninstall
 First, create the following files containing secrets used by the example cluster:
 
 - `config/example-cluster/pull-secret` a valid pull secret for image pulls.
-- `config/example-cluster/ssh-key` an SSH public key for guest node access
+- `config/example-cluster/ssh-key` an SSH public key for guest node access.
+- `config/example-cluster/aws-creds` a base64 encoded [aws credentials file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
```

Comment:
I don't think this is the right place for this secret. It's a secret required by the capa controller deployment, but these secrets are created along with an example cluster, so you won't get a running capa controller until you create the example cluster.

Reply:
Also, I don't think it needs to be base64 encoded. Kustomize should base64 encode it.

Reply:
I moved the capa controller down to the operator control. Also the secret is copied now into the target ns similar to what we do with the pull-secret.